.github/workflows/apply.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Terraform apply'
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        description: The environment to deploy against.
+        type: string
+        default: intg
+permissions:
+  id-token: write
+  contents: write
+jobs:
+  apply:
+    uses: nationalarchives/tdr-github-actions/.github/workflows/terraform_apply.yml@use-working-directory-for-apply
+    with:
+      repo-name: tna-custodian
+      environment: ${{ github.event.inputs.environment }}
+      working-directory: terraform
+    secrets:
```

Comment:
Does custodian not run against the sandbox, so needs the sandbox account number? Could be mistaken.

Reply:
Yeah good point. I've added it in here and updated https://github.com/nationalarchives/tdr-terraform-backend/pull/182 with the account number.